src/main/java/cosmetics/RemoveEffects.java

```diff
@@ -11,9 +11,9 @@
 
 public class RemoveEffects {
     
-    private Cosmetics plugin;
+    private final Cosmetics plugin;
     public RemoveEffects(Cosmetics b) {
-        plugin = b;
+        this.plugin = b;
     }
 
     public void ClearEffects(Player player) {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.